it/v1/endpoints/RetrievePensionsChargesControllerISpec.scala

```diff
@@ -29,16 +29,29 @@ import v1.stubs.{AuditStub, AuthStub, DownstreamStub, MtdIdLookupStub}
 
 class RetrievePensionsChargesControllerISpec extends IntegrationBaseSpec {
 
-  private trait Test {
+  private trait NonTysTest extends Test {
```

Comment:
For the purpose of readability I think we should define `NonTysTest` and `TysIfsTest` after defining `Test` (line 44)

Reply:
hmm I prefer the implementation class/trait before the inheritance one.
Normally you do not need to see the logic of the parent class does. It should be encapsulated.
Happy to change if its the HMRC way of doing things